gui/default/syncthing/core/syncthingController.js

```diff
@@ -47,7 +47,6 @@ angular.module('syncthing.core')
         $scope.neededPageSize = 10;
         $scope.failed = {};
         $scope.failedCurrentPage = 1;
-        $scope.failedCurrentFolder = undefined;
```

Comment:
You requested to not include https://github.com/syncthing/syncthing/pull/3986/commits/8added2f637fcc380290d0ca7529688594c99cd7#diff-10f45c616bfdd371810aecdd203d0f36L56 as initializing an object as undefined is unnecessary, so I removed this instance of the same thing as well - isn't really related to this PR though, so I could separate it. I just think it should be consistent, either use this as a "list" including undefined objects or just initialize what is really needed.